### ISSUE TITLE
can't use --seed-hosts without --auto-hosts

### ISSUE THREAD

**Issue by dlenski** [sshuttle/sshuttle#94]
I'd like to be able to access a few specific whitelisted hosts from inside the VPN by name, but *not* to automatically add any other hostnames inside the VPN, since they may conflict with other hostnames in my local search domain.

I was hoping that I could use `--seed-hosts` without `--auto-hosts`/`-H` to accomplish this:

```bash
$ sshuttle --seed-hosts host1,host2,host3 -r user@gateway 10.0.0.0/24
error: --seed-hosts only works if you also use -H
```

Is there a technical reason why I can't use `--seed-hosts` without also using `--auto-hosts`?

Basically, I'd like to be able to turn off the `--auto-hosts` behavior after scanning for the initial list.

**Comment by brianmay** [comment]
I am not very familiar with either option. I believe the idea is that the ``--auto-host`` option starts a hostwatch process on the server that watches for {host,ip} pair and then sends them to the client. I also believe that ``--seed-hosts`` is to provide a hint to the hostwatch process as to what hosts it should look for on initial start.

So as written, it is not possible to start the hostwatch process required for ``--seed-hosts`` without it also performing the duties of ``--auto-hosts``.

Is there any particular reason why you want to behaviour of ``--seed-hosts`` without the behaviour of the ``--auto-hosts``?

**Comment by vieira** [comment]
I am not a user of this feature myself but from what I understand @dlenski would like that `host1,2,3` are looked up so he can access them by name but no other lookups are made. 

With `--auto-hosts` hostwatch will be more proactive and in addition to "resolving" the names passed with `--seed-hosts` it will also lookup names in `/etc/hosts` in the server, reverse lookup ips in `netstat` and even poll local SMB servers for hosts.

So I guess there is a case for wanting one but not the other, however as I use neither I am not too sure.

**Comment by dlenski** [comment]
@vieira, you are exactly right and this is a working solution for me. I looked at [the section of `hostwatch.py` which you modified](https://github.com/vieira/sshuttle/blob/495b3c39eac62b855d24652d4b657d8714727d78/sshuttle/hostwatch.py#L270-L277) but wasn't sure if this was a sufficient change.

You identified exactly what I am trying to do: I want to add a few specific hostnames to `/etc/hosts` if they are found, but not continue to scrape any and all new hostnames that appear. (I am frequently connected to several VPNs simultaneously, and don't want to include any hostnames other than the ones I've explicitly whitelisted, to avoid any collisions.)

**Comment by brianmay** [comment]
Fixed in #95.
